### PR TITLE
Ret 1464

### PR DESCRIPTION
**Pull request by lvo700**
https://tools.hmcts.net/jira/browse/RET-1464

Added employment details complex types.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: definitions/json/EventToComplexTypes.json [hunk 1]
```diff
@@ -1061,6 +1160,105 @@
     "FieldDisplayOrder": 7,
     "DisplayContext": "MANDATORY"
   },
+  {
+    "ID": "CreateCase",
+    "CaseEventID": "initiateCaseDraft",
```
Comment (marcjudge) [inline definitions/json/EventToComplexTypes.json:1165]: You can remove all these with 'initiateCaseDraft'

Comment (lvo700) [reply definitions/json/EventToComplexTypes.json]: Removed the initiateCaseDraft events

File: yarn.lock [hunk 2]
```diff
@@ -3,1305 +3,1300 @@
...
-    debug "^4.3.2"
-    espree "^9.3.1"
-    globals "^13.9.0"
-    ignore "^4.0.6"
-    import-fresh "^3.2.1"
-    js-yaml "^4.1.0"
-    minimatch "^3.0.4"
-    strip-json-comments "^3.1.1"
+  "integrity" "sha512-C1DfL7XX4nPqGd6jcP01W9pVM1HYCuUkFk1432D7F0v3JSlUIeOYn9oCoi3eoLZ+iwBSb29BMFxxny0YrrEZqg=="
+  "resolved" "https://registry.npmjs.org/@eslint/eslintrc/-/eslintrc-1.1.0.tgz"
```
Comment (marcjudge) [inline yarn.lock:7]: change registry back to yarn

Comment (lvo700) [reply yarn.lock:7]: Changed back to yarnpkg

**Review by marcjudge — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**